jenkins/check-for-build.jenkinsfile

```diff
@@ -81,6 +82,8 @@ pipeline {
                                     build job: "${TARGET_JOB_NAME}", parameters: [
                                     string(name: 'INPUT_MANIFEST', value: "${INPUT_MANIFEST}")
                                     ], wait: true
+                                } else {
+                                    throw new Exception("Job ${TARGET_JOB_NAME} is not a valid option")
```

Comment:
Use `error` [doc](https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#error-error-signal) to make sure the build is marked a failure.  I am not a fan of exceptions as standard flow of control mechanism.

Reply:
Fixed in latest commit by checking the job name matches the 2 currently-supported jobs (`distribution-build-opensearch`, `distribution-build-opensearch-dashboards`